src/NUnitEngine/nunit.engine/Services/DefaultTestRunnerFactory.cs

```diff
@@ -57,6 +57,7 @@ public override ITestEngineRunner MakeTestRunner(TestPackage package)
             // then defer to the AggregatingTestRunner, which will make
             // the decision on a file by file basis so that each project
             // runs with its own settings.
+            // TODO: What about when ProcessModel is Parallel?
```

Comment:
I'm sure how common this edge case would be, but if you specify a project and an assembly they will use the AggregateTestRunner and run serially instead of in Parallel. When specifying `Parallel` I would expect all assemblies to run in parallel, regardless if a project is included in the list. 

https://github.com/chris-smith-zocdoc/nunit/blob/ParallelProcessRunner/src/NUnitEngine/nunit.engine.tests/Services/DefaultTestRunnerFactoryTests.cs#L83